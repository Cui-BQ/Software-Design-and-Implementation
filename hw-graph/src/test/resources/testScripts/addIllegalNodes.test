# Test add Illegal node, e.g. node = null

# Create a graph
CreateGraph graph1

# Add null
AddNode graph1 null
