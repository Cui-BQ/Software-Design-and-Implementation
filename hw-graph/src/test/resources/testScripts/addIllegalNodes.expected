# Test add Illegal node, e.g. node = null

# Create a graph
created graph graph1

# Add null
Exception: java.lang.IllegalArgumentException: node is null